### ISSUE TITLE
[RyuJIT/arm64][JitStress=1] Assertion failed 'interval->physReg != REG_NA && interval->physReg != REG_STK'

### ISSUE THREAD

**Issue by BruceForstall** [dotnet/coreclr#15036]
With x64_arm64_altjit:

```
set COMPlus_JitStress=1

set COMPlus_AltJit=*
set COMPlus_AltJitNgen=*
set COMPlus_AltJitName=protononjit.dll
set COMPlus_NoGuiOnAssert=1
set COMPlus_AltJitAssertOnNYI=1
set COMPlus_AltJitSkipOnAssert=0
set COMPlus_ContinueOnAssert=0
```

failure:

```
Assert failure(PID 3976 [0x00000f88], Thread: 13768 [0x35c8]): Assertion failed 'interval->physReg != REG_NA && interval->physReg != REG_STK' in 'Gen`1[__Canon][System.__Canon]:Target(ref):this' (IL size 158)

    File: e:\gh\coreclr2\src\jit\lsra.cpp Line: 7658
    Image: E:\gh\coreclr2\bin\tests\Windows_NT.x64.checked\Tests\Core_Root\CoreRun.exe
```

Test:
```
baseservices\threading\generics\TimerCallback\tighttimercallback\tighttimercallback.cmd 
```

https://ci.dot.net/job/dotnet_coreclr/job/master/job/jitstress/job/x64_arm64_altjit_checked_windows_nt_jitstress1_prtest/12/consoleText

Also, JitStress=2.

**Comment by BruceForstall** [comment]
@dotnet/arm64-contrib @CarolEidt

**Comment by CarolEidt** [comment]
Fixed with #16422
